docs/cow-protocol/reference/core/auctions/rewards.md

```diff
@@ -0,0 +1,83 @@
+
+Moreover, the score a solver reports is required to be smaller than the solution's $$\textrm{successfulQuality}$$, defined as the $$\textrm{observedQuality}$$ when the solution successfully executes. Since solvers are currently submitting the full solution along with their score, this is very easily checked; if the solution successfully settles on chain, its $$\textrm{successfulQuality}$$ is equal to the sum of the total surplus generated and the fees collected, and both are fully determined from the set of orders the solution intends to execute and the clearing price vector it reports.
+
+:::note
+
+When including the gas costs of executing a transaction, the winning solver's payoff is bounded above by $$c$$. However, it is not bounded below because, in case of a revert, the solver pays at most $$c$$ to the protocol and may also incur gas costs.
+
+:::
+
+To limit currency mismatch, the solver receives $$\min(\textrm{payment}, \textrm{observedCost})$$ in `ETH` and $$\textrm{payment} - \min(\textrm{payment}, \textrm{observedCost})$$ in `COW`, using an up-to-date price. In other words, the solver payment is split into a gas reimbursement paid in `ETH` and a reward paid in `COW`. To arrive at the budget for consistency rewards, these per-auction rewards paid in `COW` are deducted from the total rewards budget.
+
+Finally, the winning solver may pay additional costs, such as, for example, negative slippage once a solution is implemented. These costs are not an explicit element of the mechanism, but they are relevant in determining the solver's optimal strategy. More precisely, per [CIP-17](https://snapshot.org/#/cow.eth/proposal/0xf9c98a2710dc72c906bbeab9b8fe169c1ed2e9af6a67776cc29b8b4eb44d0fb2), solvers are responsible for managing potential slippage incurred by the settlements they settle. This is a component that affects payouts, but can be treated completely separately, and we do so in [slippage accounting](slippage).
+
+## Solver bidding strategies
+
+Apart from submitting their solutions, solvers must decide on a score to submit as a bid in the auction. In general, this score can be chosen freely by the solver (within the restriction $$0 < \textrm{score} < \textrm{successfulQuality}$$).
+
+Let $$p$$ the probability that a solution successfully executes, let $$SC$$ be the costs that a solver pays if only if a solution successfully executes, and $$C$$ the costs that a solver pays independently of whether a solution executes. For example, variable costs could include potential negative slippage of an executed solution, while fixed costs could include gas costs that arise even for a reverted solution.
+
+Ignoring capping, the winning solver's expected payoff is:
```

Comment:
Maybe mention the easier to understand intuition that due to the cost from reverting settlements the expected quality should be discounted by the the revert probability in order to reach a score that leads to a positive payout in expectation. The cap is what makes everything messy.

Reply:
I do believe that this was reviewed and adjusted in PR #28 to take this into account.